package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Hey Aaron,
How did this get added to the package-lock? The package required to run our app is the dir /front_end_project/